pedra_papel_tesoura/spec/pedra_papel_tesoura_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_base'
+require 'pedra_papel_tesoura'
+
+describe 'Pedra, Papel e Tesoura' do
+  it 'deve retornar o cenário de derrota do jogador quando usa pedra contra papel' do
```

Comment:
Mano, que acha de agrupar os cenários por contextos aqui? Por exemplo:
```suggestion
  context 'deve retornar o cenário de derrota do jogador' do
    it 'quando usa pedra contra papel' do
    end
  end
```

Reply:
Contexto adicionado no commit [f233484](https://github.com/vindi/transforma-lab/pull/2/commits/f233484717fe56424d645ddc45ed9bcef65587d9)